### PR TITLE
update readme to add local dev postgres environment instruction

### PR DESCRIPTION
**Pull request by uwaiszaki**
Update readme for the fix of #1479.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ The backend server is made up of a postgres database and an associated API that
   - `TM_SECRET`=define-freely-any-number-and-letter-combination
   - `TM_CONSUMER_KEY`=oauth-consumer-key-from-openstreetmap
   - `TM_CONSUMER_SECRET`=oauth-consumer-secret-key-from-openstreetmap
+* Adjust server/config.py to set `POSTGRES_ENDPOINT = os.getenv('POSTGRES_ENDPOINT', 'localhost')`
```
Comment (xamanu) [inline README.md]: I think the instructions are not the case for all kind of setups. And I don't want users to encourage changing the `config.py`. Woudn't the setup work with a `POSTGRES_ENDPOINT=localhost`? And maybe find a phrase to be more open to any kind of setup.

Comment (uwaiszaki) [reply README.md]: @xamanu I have updated the pr.

**Comment by xamanu** [comment]
This should be moved to `docs/setup-development.md`.

**Comment by JorgeMartinezG** [comment]
@uwaiszaki could you pleas include POSTGRES_PORT variable also?

**Comment by uwaiszaki** [comment]
@xamanu @JorgeMartinezG I have updated the pr, please give your reviews. Thanks

**Pull request merged**